lib/Yam/agama/agama_base.pm

```diff
@@ -35,4 +36,9 @@ sub test_flags {
     return {fatal => 1};
 }
 
+sub upload_agama_traces {
```

Comment:
neatpicking, could you call it `upload_traces()` they are not really agama traces, they are playwright traces for agama, and also it is more uniform with other names like upload_logs.

Reply:
Renamed to `upload_traces`